tests/testthat/test-pie.R

```diff
@@ -279,6 +279,6 @@ test_that("pie.dt() returns correct information about missing data", {
   # number of completeCases should be <= complete cases for each var
   expect_equal(all(attr(dt, 'completeCasesAllVars')[1] <= completecasestable$completeCases), TRUE)
   expect_equal(attr(dt, 'completeCasesAxesVars')[1] >= attr(dt, 'completeCasesAllVars')[1], TRUE)
-  dt <- pie.dt(df, map, value='count', evilMode = TRUE)
+  dt <- pie.dt(df, map, value='count', evilMode = 'strataVariables')
```

Comment:
For pie do we need to test 'allVariables' since the map will have evilMode for the x axis, assuming that's still the plan?